**************************************************************************************************************
6.0.41-hotfix1 (2014/11/28)


--------------------------------------------------------------------------------------------------------------
ganga/python/GangaAtlas
--------------------------------------------------------------------------------------------------------------
* Update to DQ2Clients 2.6.1_rc6
* Introduce rucio-clients 0.2.5 external
* Update all DQ2 interactions in DQ2Dataset and Panda to use rucio compatible calls
* Use DQ2(force_backend='rucio')


--------------------------------------------------------------------------------------------------------------
ganga/python/GangaPanda
--------------------------------------------------------------------------------------------------------------
* Update all DQ2 interactions in Panda to use rucio compatible calls

**************************************************************************************************************
